lib/OpenQA/Log.pm

```diff
@@ -213,7 +214,7 @@ sub redact_settings ($vars) {
 
 sub redact_settings_in_file ($file) {
     $file = path($file);
-    $file->spurt(encode_json(redact_settings(decode_json($file->slurp))));
+    $file->spurt(Cpanel::JSON::XS->new->pretty->encode(redact_settings(decode_json($file->slurp))));
```

Comment:
I would have liked to read more information within the git commit message about how this fixed stuff. What is wrong with Mojo::JSON?

Reply:
It did not nicely format the output. I hoped the commit message would imply that. I've been switching to Cpanel here for the best consistency with os-autoinst and I'm not even sure whether Mojo support pretty formatting .